submissions/adeyemiJuwon/javascript/loop.js

```diff
@@ -0,0 +1,19 @@
+let a = 123;
+  let b = 7;
+  let c = a - b;
+
+for (var i = 1; i < c; i++){
+
+    if (i % 3 == 0 && i % 5 == 0){
+        console.log('Software Developr');
```

Comment:
There is a typo in `developer` @Samju200

Reply:
Done 
Thank you @ToluAkin